dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -12,50 +12,56 @@ namespace winrt
 
 namespace winrt::Microsoft::Windows::PushNotifications::implementation
 {
+    winrt::TypedEventHandler<
+        Microsoft::Windows::PushNotifications::PushNotificationChannel,
+        Microsoft::Windows::PushNotifications::PushNotificationReceivedEventArgs> s_typedEventHandler;
+    wil::srwlock s_lock;
```

Comment:
It would be good having a comment explaining why these need to be static variables.